teps/0098-workflows.md

```diff
@@ -256,11 +297,160 @@ Why should this TEP _not_ be implemented?
+rather than building extensibility mechanisms directly into the Workflows API.
+The conformance spec would be based on the work done by [existing projects](#prior-art).
+Platform builders would create their own implementations of the Workflows API to support customized
+connections to different SCMs or customized secrets storage. The Tekton team would maintain an implementation of the API that supports only
+the most commonly used SCMs.
+
+Pros:
+- App developers can run their Workflows on different platforms without having to rewrite them
+- Platform builders can also introduce specific features that make the most sense for their platform
+- Existing implementations of E2E CI/CD solutions build on top of Tekton can work to become Workflows conformant
+as the reference implementation evolves.
+
+Cons:
+- It will be difficult to come up with a Workflows conformance spec without first creating a reference implementation
+that supports several SCMs. It's unclear whether designing a good conformance spec is easier than designing a good
+API extensibility mechanism (and later on declaring a conformance spec).
+- Platform builders who want to change only part of the implementation will be forced to recreate the entire implementation.
+  - For example, a platform builder might want to add some custom steps to the process of connecting to a repo, or add their own backing
+  for secrets storage, without rewriting much of the project.
+
+### Adopt Pipelines as Code as Workflows Replacement
+
+In this solution, originally proposed in [#866](https://github.com/tektoncd/community/issues/883),
+Pipelines as Code would be transferred to the tektoncd organization,
+built and released as a Tekton project, and likely renamed to "Workflows", since the scope
+of the Workflows project is larger than storing Pipelines in version control.
+The Tekton community would maintain the project and shape its future to meet the goals of the Workflows project.
+The doc [Pipelines as Code -> Tekton Workflows](https://docs.google.com/document/d/1JAnXFkvs4aeciAJEsl7454e0J3ZHbL1yCPncMBHlTtM/)
+compares Pipelines as Code to the goals of the Workflows project.
+
+Suggested steps:
+1. Move the project to the tektoncd org, rename it, run tests on Tekton infrastructure, and publish releases to the tekton-releases GCP project.
+Use tekton standards for code, conduct, and contributions. Choose and document a Tekton stability level for the existing API.
+2. Update the project to make more extensive use of remote resolution. For example, we may want to keep the ability to refer to Tekton CRDs in the same
+repo using local paths, but encourage the use of the built-in hub resolvers rather than using the Pipelines as Code
+[hub resolution](https://pipelinesascode.com/docs/guide/resolver/#tekton-hubhttpshubtektondev) and the built-in git resolver rather than using
+the Pipelines as Code [remote URL syntax](https://pipelinesascode.com/docs/guide/resolver/#remote-http-url).
```

Comment:
Not that this is also the route we want to take 🙃 
Because we already have a user base, we need to support the current way for a while, *but* we want for our users to rely more and more on remote resolution as this is the "most" tekton native way 🙃

Reply:
added a note about this, thanks!